packages/main/src/controllers/credentials/CredentialTypeController.ts

```diff
@@ -57,13 +60,14 @@ export class CredentialTypesController {
         const schemaResult = await agent.modules.anoncreds.getSchema(record.credentialDefinition.schemaId)
 
         const schema = schemaResult.schema
+        const revocationRegistryIds = record.getTag('revocationRegistryDefinitionId') as string
 
         return {
           id: record.credentialDefinitionId,
           name: (record.getTag('name') as string) ?? schema?.name,
           version: (record.getTag('version') as string) ?? schema?.version,
           attributes: schema?.attrNames || [],
-          revocationId: record.getTag('revocationDefinitionId') as string,
+          revocationId: revocationRegistryIds ? revocationRegistryIds.split('::') : undefined,
```

Comment:
What's this `split('::')` used for?

Reply:
Fixed: take the revocationRegistryDefinitionIds from AnonCredsRevocationRegistryDefinitionRepository and send for array